apps/i18n/common/en_us.json

```diff
@@ -844,6 +844,7 @@
   "errorFindingClassLibraries": "Unable to load your class libraries at this time. Please check your internet connection and try again.",
   "errorIncompleteBlockInFunction": "Click \"edit\" to make sure you don't have any blocks missing inside your function definition.",
   "errorGenericLintError": "Your program contains an editor warning that needs to be corrected. Hover over the icons near the line numbers in the editor to learn more.",
+  "errorLoadingDocumentation": "An error occurred while loading documentation. Please try again.",
```

Comment:
nit: is there a refresh button? or does the user need to reload the level?

Reply:
There's no refresh button They would need to reload the tab (switch tabs and go back again), wasn't sure how to message that. We could just say "an error occurred..." without "Please try again"

Reply:
Yeah it's definitely not a big deal. I was thinking something like "please reload the page" but idk if we're worried that might lead to lost work?

Reply:
The autosave is pretty robust so I wouldn't worry about that. It's just slow to reload the page

Reply:
Up to you! I was only asking because it sounds like a call to press a button or something but I might be overthinking it :)